packages/cardano-services-client/src/HttpProvider.ts

```diff
@@ -1,9 +1,12 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import { HttpProviderConfigPaths, Provider, ProviderError, ProviderFailure } from '@cardano-sdk/core';
 import { Logger } from 'ts-log';
+import { apiVersion } from './version';
 import { fromSerializableObject, toSerializableObject } from '@cardano-sdk/util';
-import { version } from './version';
 import axios, { AxiosAdapter, AxiosRequestConfig, AxiosResponseTransformer } from 'axios';
+import path from 'path';
+
+const packageJson = require(path.join(__dirname, '..', 'package.json'));
```

Comment:
I think that won't work with built package, because it's more nested: `dist/cjs/HttpProvider.js` would resolve to `dist/cjs`

Reply:
`path.join(__dirname, '..', 'package.json')` resolves to `dist/package.json`

Reply:
It's different when it's running with `ts-node` from `src/HttpProvider.ts`